src/main/java/org/jetbrains/research/refactorinsight/folding/JavaFileFinder.java

```diff
@@ -0,0 +1,81 @@
+package org.jetbrains.research.refactorinsight.folding;
+
+import com.intellij.psi.*;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Arrays;
+
+public class JavaFileFinder {
```

Comment:
The name is not clear. We can create `PsiUtils` class in the `util` package and collect there all methods that relate to extracting some things from PSI as we do it in `findMethod` and `findClass` methods